pfcpiface/parse-far_test.go

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022-present Open Networking Foundation
+
+package main
+
+import (
+	"net"
+	"testing"
+
+	"github.com/omec-project/upf-epc/test/integration"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/wmnsk/go-pfcp/ie"
+)
+
+type testCase struct {
+	input       *ie.IE
```

Comment:
I like the refactored version. I only miss the `output`/`exptected` field. The expected object should be `far` struct filled with variables that we expect (farID, action, etc.)

Reply:
Done